Domains/4-Application/Connectors/CivilInfrastructureFramework/CifSurvey.remarks.md

```diff
@@ -0,0 +1,9 @@
+---
+remarksTarget: CifSurvey.ecschema.md
+---
+
+# CifSurvey
+
+iModel Connector schema used by the Civil iModel Connector while publishing datasets from any Civil Infrastructure Framework (CIF)-based application. The aspect classes contained within this schema aim to provide a similar user-experience with respect to properties in the source CIF application, as well as make its data available for re-purposing. 
```

Comment:
This text sounds like a copy/paste from more generic schemas such as CifCommon or CifGeometricRules. Since CifSurvey sounds more specific than those, maybe the first sentence could point that out:

_iModel Connector schema used by the Civil iModel Connector while publishing datasets with data created by Survey Tools in Bentley Civil Design applications. The aspect classes contained within this schema aim to provide a similar user-experience with respect to properties in the source Civil Design application,..._

Reply:
Updated as suggested